dispatcher/backend/src/utils/offliners.py

```diff
@@ -93,7 +93,8 @@ def simplified(value):
 def compute_flags(flags, use_equals=True):
     """flat list of params from dict of flags"""
     params: List[str] = []
-    for key, value in flags.items():
+    # flags are sorted to produce consistent ordering, especially useful for tests
```

Comment:
This is only possible by using PostgreSQL `json` datatype instead of `jsonb`. This has performance consequences, and we can't anymore create indexes on the JSON content, but this is not an issue for us (at least for now).
More details : https://www.postgresql.org/docs/current/datatype-json.html